monkey/agent_plugins/exploiters/hadoop/src/plugin.py

```diff
@@ -0,0 +1,181 @@
+"""
+    Remote code execution on HADOOP server with YARN and default settings
+    Implementation is based on code from
+    https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn
+"""
+
+import logging
+from datetime import datetime
+from typing import Any, Dict, Sequence
+from uuid import UUID
+
+# common imports
+from common import OperatingSystem
+from common.event_queue import IAgentEventPublisher
+from common.types import Event
+from common.utils.code_utils import del_key
+
+# dependencies to get rid of or internalize
+from infection_monkey.exploit import IAgentBinaryRepository
+from infection_monkey.exploit.tools.helpers import get_agent_dst_path
+from infection_monkey.exploit.tools.http_tools import HTTPTools
+from infection_monkey.exploit.tools.web_tools import build_urls
+from infection_monkey.i_puppet import ExploiterResultData
+from infection_monkey.model import (
+    HADOOP_LINUX_COMMAND,
+    HADOOP_WINDOWS_COMMAND,
```

Comment:
These are removed from the `infection_monkey.model`. Add them to the plugin and use them from there.